### ISSUE TITLE
Irace - Cannot disable MultiStart Component

### ISSUE THREAD

**Issue by isaaclo97** [mork-optimization/mork#177]
**Describe the bug**
The autoconfig creates by default all components as multistart.

```
autoconfig\src\main\java\es\urjc\etsii\grafo\autoconfig\irace\IraceOrchestrator.java
```

```
  if (this.isAutoconfigEnabled) {
      // Because autoconfig iterations executes between two time intervals, if the algorithm finishes
      // but there is extra time available, keep executing the same algorithm until the timelimit is reached.
      // If not, we could be penalizing faster simpler algorithms against more complex ones.
      int iterations = Integer.MAX_VALUE / 2;
      algorithm = new MultiStartAlgorithm<>(algorithm.getShortName(), algorithm, iterations, iterations, iterations);
      algorithm.setBuilder(this.solutionBuilder);
  }
```

**Expected behavior**
Allow the user to decide whether to have a multistart algorithm. Maybe the user wants to implement his own and not use the multistart. 

**Versions (ignore not applicable ones):**
 - Mork version: v0.17 SNAPSHOT

**Comment by rmartinsanta** [comment]
This is intentional, as we do not know beforehand the runtime of any generated algorithm. Otherwise, we would be penalizing faster algorithms, as they would not run for their corresponding allocated computational budget.

Any contributions towards improving this issue are welcome, i.e., giving more options to users, but note that this is not a bug, and I think it is a reasonable default behavior.

**Comment by rmartinsanta** [comment]
Implemented in #187, feature can now be disabled via configuration file or CLI switch (--solver.autorestart=false).

**Comment by rmartinsanta** [comment]
Implemented, and tested by @isaaclo97. Missing docs will be published soon.
